deploy/contents/sync/scrap/get_version.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+get_version() {
+    local server_address="$1"  #Server address name 
+    local token="$2" #Access key for pipe cli(API token) 
+    local output_dir="$3" #Output directory name where config-map will be saved 
+    local output_file="version" #Output file name where preferences will be saved
+    local api_endpoint='pipeline/restapi/app/info'
+
+    if [ -z "$token" ]; then
+        echo "Please provide a token"
+        exit 1
+    fi
+
+    if [ -z "$server_address" ]; then
+        echo "Please provide the server address"
+        exit 1
+    fi
+
+    
+
+    # Send GET request to the server
+    curl -H "Authorization: Bearer ${token}" "https://${server_address}/${api_endpoint}" -H "Accept: application/json" | jq -r '.payload' > $output_dir/$output_file
```

Comment:
let's split this operation:

  - First, get the output from ssh
  - Second, check that command was successful
  - Finally, do the post processing of the data